src/EditorFeatures/Core/Shared/Options/InternalFeatureOnOffOptions.cs

```diff
@@ -8,17 +8,17 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Options
 {
     internal sealed class InternalFeatureOnOffOptions
```

Comment:
>InternalFeatureOnOffOptions [](http://example.com/codeflow?start=26&length=27)

Do we still need these at all?

Reply:
I can move all of them to separate classes